pilot/pkg/config/kube/crd/types.gen.go

```diff
@@ -76,6 +76,16 @@ var KnownTypes = map[string]SchemaType{
 		},
 		Collection: &ServiceEntryList{},
 	},
+	schemas.SyntheticServiceEntry.Type: {
```

Comment:
@Nino-K Just to confirm, you didn't edit this by hand correct? Just want to make sure this update was autogenerated.

Reply:
Yeah not edited that file, I think it got generated from `pilot/tools/generate_config_crd_types.go`.